tutorials/plot_source_alignment.py

```diff
@@ -0,0 +1,95 @@
+"""
+.. _tut_source_alignment:
+
+Source alignment
+================
+
+The aim of this tutorial is to show how to visually assess that the data
+are well aligned in space for computing the forward solution.
+"""
+import os.path as op
+
+import mne
+from mne.datasets import sample
+
+print(__doc__)
+
+
+###############################################################################
+# Set parameters
+# --------------
+data_path = sample.data_path()
+subjects_dir = op.join(data_path, 'subjects')
+raw_fname = op.join(data_path, 'MEG', 'sample', 'sample_audvis_raw.fif')
+tr_fname = op.join(data_path, 'MEG', 'sample', 'sample_audvis_raw-trans.fif')
+raw = mne.io.read_raw_fif(raw_fname)
+
+
+###############################################################################
+# :func:`mne.viz.plot_alignment` is a very useful function for inspecting
+# the surface alignment before source analysis. The function automatically
+# looks for the Freesurfer surfaces from the subject's folder.
```

Comment:
This isn't what's being done (and hopefully we remove it anyway). Say explicitly something like "here we use trans=None, which (incorrectly!) equates the MRI and head coordinate frames."